app/actions/route_destination_update.rb

```diff
@@ -14,17 +14,34 @@ def update(destination, message)
 
           destination.save
         end
+
+        runners = CloudController::DependencyLocator.instance.runners
+        destination.processes.each do |process|
+          notify_backend_of_route_update(process, runners)
+        end
+
+        destination
       end
 
       private
 
+      def notify_backend_of_route_update(process, runners)
```

Comment:
The implementation looks similar to `ProcessRouteHandler`. What about reusing this handler?

Reply:
Are you suggesting reusing the `ProcessRouteHandler` entirely? Or just its `notify_backend_of_route_update` method? I don't think `ProcessRouteHandler` itself should be invoked since there is actually no work to do on the processes themselves, but I could look at modifying the `notify_backend_of_route_update` function to support both. It would just need refactored because it relies a lot on instance variable state.

Reply:
I was actually thinking about simply reusing the `ProcessRouteHandler` as done [here](https://github.com/cloudfoundry/cloud_controller_ng/blob/6d325497d5c1e1f6604456f970ce7a7bb9f8e392/app/models/runtime/route.rb#L257) and [here](https://github.com/cloudfoundry/cloud_controller_ng/blob/6d325497d5c1e1f6604456f970ce7a7bb9f8e392/app/models/services/route_binding.rb#L28).

Reply:
Makes sense, I don't know why I thought `notify_backend_of_route_update` was a private method in there.

Reply:
Much cleaner, thanks for the suggestion @philippthun !
https://github.com/cloudfoundry/cloud_controller_ng/pull/3787/commits/13f67275814d5d187fcaac118e8c815cc5e6e377